mock/cpp_mock_scenarios/launch/mock_test.launch.py

```diff
@@ -77,13 +77,27 @@ def generate_launch_description():
     scenario_package = LaunchConfiguration("package", default="cpp_mock_scenarios")
     junit_path = LaunchConfiguration("junit_path", default="/tmp/output.xunit.xml")
     launch_rviz = LaunchConfiguration("launch_rviz", default=False)
+    vehicle_model = LaunchConfiguration("vehicle_model", default="jpntaxi")
```

Comment:
jpntaxi is not public vehicle description.
In OSS perspective, it is good for default value to use a public vehicle description like lexus or sample_vehicle.

Reply:
@hakuturu583 If you use `sample_vehicle`, you should use `sample_sensor_kit` as sensor_model.